hooks/tk-multi-publish2/maya/publish_lod_geometry.py

```diff
@@ -228,6 +272,11 @@ def publish_files(self, task_settings, item, publish_path):
             "Exported group %s to '%s'." % (item.properties.fields["node"], publish_path)
         )
 
+        # Renaming top group name to be the asset name, in exported alembic.
+        published_abc = publish_path.replace("\\", "/")
+        asset_name = item.context.entity["name"]
+        self._rename_abc_top_group(str(published_abc), str(asset_name))
```

Comment:
Do we really need an explicit str conversion?

Reply:
yeah, because published_abc returns a unicode, and alembic accepts ONLY a string value, while loading an alembic archive. No need here to convert asset_name, i can remove that.

Reply:
Ah, okay.